jpf-symbc/src/main/gov/nasa/jpf/symbc/SymbolicListener.java

```diff
@@ -206,17 +206,37 @@ public void propertyViolated(Search search) {
         // }
     }
 
+    /**
+     * It parses classname and filename to fill the value of assumption.scope at violation witness
+     * Both classname and filename are needed to construct the edge of the witness
+     * @param assumptionScope is a value of assumption.scope attribute of violation witness
+     * @param fileName is a value of originfile attribute of violation witness
```

Comment:
same comment about the "@" above

Reply:
I missed to fix this after rewriting methods. I removed @param sign. Thank you!